modules/flake-parts/_template.nix

```diff
@@ -0,0 +1,8 @@
+{ self, lib, ... }: {
+  perSystem = { config, self', inputs', pkgs, ... }: {
+
+  };
+  flake = {
+
+  };
+}
```

Comment:
I think this file is unused

Reply:
That is intentional. In case someone wants to create a new flake-parts module here, they just need to copy the _template.nix.
Do you still prefer removing it?

Reply:
I don't have a preference right now. Is that a convention that exists elsewhere?

Reply:
No, it's just a personal habit with the goal of increasing my productivity when creating new modules.